vmdb/app/helpers/container_node_helper/textual_summary.rb

```diff
@@ -40,4 +40,10 @@ def textual_ems
     end
     h
   end
+
+  def textual_container_groups
+    num_of_container_groups = @record.number_of(:container_groups)
+    label = "Container groups"
+    {:label => label, :image => "container_group", :value => num_of_container_groups}
```

Comment:
@h-kataria pls see my first comment in this PR. links are not part of it, they will b added later in different change. same regarding the other PR.